### PR TITLE
Added my hand drawn tile

### PR DESCRIPTION
**Pull request by arjunpandey2003**
js updated

  http://rawgit.com/arjunpandey2003/labyrinth/master/index.html

### CODE REVIEW

**Comment by arjunpandey2003** [comment]
screen shot 2018-01-13 at 7 49 01 pm

**Comment by niccokunzmann** [comment]
Cool, thanks.

File: js/tiles.js [hunk 1]
```diff
@@ -303,6 +303,13 @@ const door = {
             this.ground = this.createImage("tiles/rooms/floor/live.svg");
             alertInfo("More to Go !", "");
         },
+    }),
+       design: Object.assign({}, OpenDoors, {
+        createImages: function() {
+            this.wallTop = this.createImage("tiles/rooms/door/top.svg");
+            this.wallRight = this.createImage("tiles/rooms/door/right.svg");
+            this.ground = this.createImage("tiles/rooms/floor/handarjun.svg");
```
Comment (niccokunzmann) [inline js/tiles.js]: Would you like to create the ground tile before the others? then maybe it is not overlapping the wall.

Comment (arjunpandey2003) [reply js/tiles.js]: sure will do

**Pull request merged**